plugins/org.yakindu.sct.doc.user/src/user-guide/overview.textile

```diff
@@ -3,89 +3,95 @@ p.
+Please also note that Harel statecharts combine the characteristics of Mealy and Moore machines, hence outputs can be produced by states as well as transitions as indicated in the statechart above.
 
-!(large)images/ferry_harel_1.png(Ferry example as a Harel statechart)!
+We can even go one step further and hide the logic of the _Motion Detection Mode_ into a sub diagram. In that way, the system gets more comprehensive as one can directly see the different modes and how to switch between them.
 
-p=. Ferry example as a Harel statechart
+!(medium)images/overview_harel_3.png(Extended light switch example as a Harel statechart with sub diagrams)!
 
-The *carEnter* / *carLeave* states were substituted by a composite state *FerryStops* with a substate machine that describes the behavior of the ferry at harbor. The lifts to the car deck shall be locked, because passengers must be there during transfer. There are transitions crossing the hierarchy from the substate machine to the upper-level states (inter-level transitions) like the one from state *LockLifts* to *Northwards* .
+Further concepts like orthogonality or history states are left out here for brevity. You can read about them in our "quick reference":../user-guide/quick_ref.html.
 
-h2(#overview_uml). The present age: UML state machines
 
-UML state machines are based on the statechart notation introduced by David Harel. Furthermore, the UML extends the notation of Harel statecharts by object-oriented principles. Mapping this to our ferry example, in UML we can model the possible actions of the ferry as an interface with operations stop, moveNorthwards, moveSouthwards and so on.
+*The present age: UML state machines*
+
+UML state machines are based on the statechart notation introduced by David Harel. Furthermore, the UML extends the notation of Harel statecharts by object-oriented principles. Mapping this to our light switch example, in UML we can model the possible actions of the light switch as a type with operations @turnOn()@, @turnOff()@, @setBrightness(value)@ and so on.
 
 The following table illustrates the differences between the previously described types at a glance:
 
 !(large)images/statemachine_types.png(Differences between state machine types)!
 
 p=. Differences between the state machine types
 
-The examples of this article were designed with the YAKINDU Statechart Tools, whose documentation you are reading just now.
+The examples of this article were designed with "YAKINDU Statechart Tools":https://www.itemis.com/en/yakindu/state-machine/, whose documentation you are reading just now.
```

Comment:
Mention to what category YSCT belongs. YAKINDU statecharts are based on Harel stetcharts and are very close but not identical to UML state machines. The concrete difference are explained in the documentation where they exist.

Reply:
Simulation, executable state machines...